dockerversion/release.go

```diff
@@ -0,0 +1,25 @@
+package dockerversion
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/docker/docker/api/types"
+)
+
+// DockerReleaseFile represents the content of the docker-release file
+type DockerReleaseFile struct {
+	FormatVersion string
+	DockerVersion types.DockerVersion
+}
+
+// DockerRelease returns the DockerReleaseFile
+func DockerRelease() *DockerReleaseFile {
+	if dockerReleaseFile, err := os.Open(dockerReleasePath); err == nil {
```

Comment:
Does this actually work on Windows when `dockerReleasePath` is relative, not absolute? Where is the actual `docker-release` file located? Under the root directory of the daemon? In a fixed location? And what creates it? The D4W installer?